javascripts/discourse/templates/mobile/modal/login.hbs

```diff
@@ -0,0 +1 @@
+../../modal/login.hbs
```

Comment:
Wow! Didn't know about this one.
What you do here is to say "use the modal login file we have at X path?"
Why not to use handlebars syntax such as `{{login}}` for instance?

Reply:
This is a symlink to the file, I couldn't find an elegant way to override modals on both sides since this is part of a modal component